pkg/scorecard/lib/test_definitions.go

```diff
@@ -0,0 +1,107 @@
+// Copyright 2019 The Operator-SDK Authors
```

Comment:
Nit: Any reason to include `lib` in the package path? Can we move this to `pkg/scorecard/test_definitions.go`

Reply:
I made the lib directory because I was planning to add the JSON/K8S style structs to `pkg/scorecard/apis`.

Reply:
I would also prefer if this moved to `pkg/scorecard`, think the `pkg` already implies the lib part. Haven't come across many go projects that use the `lib` directory. I would do it the other way around and move non exposed code into `internal` dir.

Reply:
@LiliC All the non-exposed scorecard code (which is most of the scorecard code) was moved to the `internal` directory in the PR. The `test_definitions.go` file just contains the structs we are using to define the tests, test results, test suites, and a couple of helpers that other users may wish to use if they are using go to make their plugins.

I though about the directory structure a bit more, and for the exposed scorecard structs/helpers I'll use `pkg/scorecard`, and for the K8S API code (which I was originally planning to put in `pkg/scorecard/apis/v1alpha1`) I will use `pkg/apis/scorecard/v1alpha1`, which is more in line with how we define K8S structs in our generated code as well.